CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Mapbox welcomes participation and contributions from everyone.
 #### Features
 - Added a new `RailwayCrossing` type to `RoadObject`s. [#5552](https://github.com/mapbox/mapbox-navigation-android/pull/5552)
 - Added point exclusion option for onboard router. [#5552](https://github.com/mapbox/mapbox-navigation-android/pull/5552)
+- Added ability to replay route alternatives with the `ReplayProgressObserver`. [#5586](https://github.com/mapbox/mapbox-navigation-android/pull/5586)
```

Comment:
```suggestion
- Improved capabilities to replay route alternatives with the `ReplayProgressObserver` by respecting the current distance traveled, without the jump to the beginning of the alternative route upon the switch. [#5586](https://github.com/mapbox/mapbox-navigation-android/pull/5586)
```

Let's clarify what exactly we're improving. The ability to replay alternatives was always there, just a little bit clunky.